src/08-accessing.js

```diff
@@ -148,7 +169,10 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  let fullAddress = person.address.street + ' ' + person.address.unit + ', ' + person.address.city + ', ' + person.address.state + ', ' + person.address.zip
```

Comment:
The better move here is to use a template literal with backticks.  This lets us spread our string across multiple lines of code.  This makes it much easier to read.